sidechain/consensus/common/src/block_import_confirmation_handler.rs

```diff
@@ -0,0 +1,122 @@
+/*
+	Copyright 2021 Integritee AG and Supercomputing Systems AG
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+
+*/
+
+use crate::error::{Error, Result};
+use itc_parentchain_light_client::{
+	concurrent_access::ValidatorAccess, BlockNumberOps, ExtrinsicSender, NumberFor,
+};
+use itp_extrinsics_factory::CreateExtrinsics;
+use itp_node_api_metadata::pallet_sidechain::SidechainCallIndexes;
+use itp_node_api_metadata_provider::AccessNodeMetadata;
+use itp_types::{OpaqueCall, ShardIdentifier};
+use log::*;
+use sidechain_primitives::traits::Header as HeaderTrait;
+use sp_runtime::traits::Block as ParentchainBlockTrait;
+use std::{marker::PhantomData, sync::Arc};
+
+/// Trait to confirm a sidechain block import. Typically this is sent to the parentchain.
```

Comment:
I'm proposing to add this explanation (parentchain part) to the Handler struct itself.
```suggestion
/// Trait to confirm a sidechain block import.
```